data-base/db_main_test.go

```diff
@@ -14,36 +16,36 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var db *RecordDB
-var records = []repo.Record{
+var db *Database
```

Comment:
do not use global variables for modification, you must init new instance of DB in **EVERY** test case